util/stream.go

```diff
@@ -144,16 +147,20 @@ func (m *MessageStream) inbound() {
 // Parse incoming message
 func (m *MessageStream) parse() {
 	for {
-		b := <-m.pool.Full
-		log.Debugf("Rcvd: %v", b.Bytes())
-		msg, err := m.parser.Parse(b.Bytes())
-		// Log all message parsing errors.
-		if err != nil {
-			log.Print(err)
-		}
+		select {
+		case b := <-m.pool.Full:
+			log.Debugf("Rcvd: %v", b.Bytes())
```

Comment:
@rhim: You're right. This was logging the whole content of the received OpenFlow message. I've modified this to happen only when an error is encountered during parsing.

My initial goal was to change only what was required to solve these problems. I intend to do more refactoring later.